products/extensions/subscriptions/concept.md

```diff
@@ -1,29 +1,137 @@
-# Concepts
+# Subscription concept
 
-To use subscriptions, you will need to be familiar with two core concepts in subscription models.
+To use subscriptions, you will need to be familiar with three core concepts: **Subscription Plans**, **Subscription Intervals**, and the **Checkout Processes**.
 
-## Plans
+## Terminology
 
-A plan is a set of rules that define the subscription. This includes the billing interval and the product that the customer will receive. Multiple intervals can be assigned to a single plan. Plans can be created and managed in the Shopware administration.
+### Subscription Plans
 
-## Intervals
+A plan is a set of rules that define the subscription.
+This includes the [billing interval](#subscription-intervals) and the product that the customer will receive.
+Multiple intervals can be assigned to a single plan.
+Plans can be created and managed in the Shopware administration.
 
-An interval is the time between each delivery cycle. For example, a delivery can be monthly, quarterly, or annually. Billing is triggered each time a delivery cycle repeats. The interval is defined in the plan and can be set to any time frame. This is also created and managed in the Shopware administration.
+### Subscription Intervals
+
+An interval is the time between each delivery cycle.
+For example, a delivery can be monthly, quarterly, or annually.
+Billing is triggered each time a delivery cycle repeats.
+The interval is defined in the plan and can be set to any time frame.
+This is also created and managed in the Shopware administration.
 
 Intervals can be of three different types:
 
-### Relative
+1. **Relative**  
+   A relative interval is determined by a previous interval.
+   For example, if a customer subscribes to a monthly plan, the next interval will be one month after the first delivery.
+   These intervals are determined using PHP's `DateInterval`.
+
+2. **Absolute**  
+   An absolute interval is determined by a fixed date.
+   For example, if a customer subscribes to a monthly plan, the next interval will be on a fixed day like the 1st or 15th of each month.
+   These intervals are defined using cron expressions.
+
+   Absolute intervals can also possess a relative element, such as a delivery occurring every 12 weeks (relative component), but exclusively on Fridays (absolute component).
```

Comment:
Is there a key `3.` missing here?

Reply:
No, I changed it after Max's review [here](https://github.com/shopware/docs/pull/1968#discussion_r2452082617) and I agree with him